proto/tendermint/abci/types.proto

```diff
@@ -533,11 +533,11 @@ message Snapshot {
 
 message TimeoutInfo {
   google.protobuf.Duration timeout_propose           = 1 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
-  google.protobuf.Duration timeout_propose_delta     = 2 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
+  google.protobuf.Duration timeout_commit            = 2 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
```

Comment:
damn! I could swear I purposefully avoided modifying the proto in a breaking way, idk how it slipped through